modules/plot/samples/plot_demo.cpp

```diff
@@ -6,10 +6,10 @@ using namespace cv;
 
 int main()
 {
-    Mat data_x(1, 50, CV_64F);
-    Mat data_y(1, 50, CV_64F);
+    Mat data_x(1, 51, CV_64F);
+    Mat data_y(1, 51, CV_64F);
 
-    for (int i = 0; i < 50; i++)
+    for (int i = 0; i < 51; i++)
     {
```

Comment:
may be  for (int i = 0; i <data_x.cols; i++) is better

Reply:
@LaurentBerger you are right. let me change it asap.